docs/octopus-rest-api/octopus-cli/list-machines.md

```diff
@@ -87,6 +87,31 @@ Common options:
                              fatal. Defaults to 'debug'.
 ```
 
+## Basic example
+This example will list all machines on the instance
+
+```text
+octo.ext list-machines --server="https://MyOctopusServer" --apiKey="API-MyApiKey"
```

Comment:
typo - `ext` instead of `exe`.

Can we not have the extension at all? It excludes linux users, and isn't really required for windows anyway

Reply:
Yeah, I can do that.